ipatests/test_integration/test_replica_promotion.py

```diff
@@ -52,8 +52,7 @@ def test_promotion_disabled(self):
                 '--ip-address', client.ip]
         result = client.run_command(args, raiseonerr=False)
         assert_error(result,
-                     'You must provide a file generated by ipa-replica-prepare'
-                     ' to create a replica when the domain is at level 0', 1)
+                     'You used the wrong mechanism to install a replica in domain level 0', 1)
```

Comment:
Have you tested this? I am pretty sure that the text returned would be much longer.